File: legacy/legacy-apis/Psi.Models.ClientConfigurationModels/Address/MultipleAddressesPage.cs
using System.Collections.Generic;

namespace Psi.Data.Models.ClientConfigurationModels.Address
{
    public class MultipleAddressesPage : SettingsBaseHelper
    {
        public MultipleAddressesPage(ISettingsBase settingsBase) : base(settingsBase)
        {
        }

        [SettingKey("MultipleAddresses.MvcPage.Enabled")]
        public bool Enabled
        {
            get { return GetBoolValue(); }
            set { SetValue(value); }
        }

        [SettingKey("MultipleAddresses.MvcPage.MinVersion")]
        public double MinVersion
        {
            get { return GetDoubleValue(); }
            set { SetValue(value); }
        }

        [SettingKey("MultipleAddresses.MvcPage.DeleteAddressEnabled")]
        public bool DeleteAddressEnabled
        {
            get { return GetBoolValue(); }
            set { SetValue(value); }
        }

        [SettingKey("MultipleAddresses.MvcPage.AddressTypesThatCanBeDeleted")]
        public List<string> AddressTypesThatCanBeDeleted
        {
            get { return GetListValue(); }
            set { SetValue(value); }
        }
    }
}

File: workspace/legacy-analyzer/output/infrastructure/models/Address/MultipleAddressesPage.ts
// Generated imports
import { AddressTypesThatCanBeDeleted } from '../AddressTypesThatCanBeDeleted';

export interface MultipleAddressesPage {
    /** @settingKey MultipleAddresses.MvcPage.Enabled */
    enabled: boolean;
    /** @settingKey MultipleAddresses.MvcPage.MinVersion */
    minVersion: number;
    /** @settingKey MultipleAddresses.MvcPage.DeleteAddressEnabled */
    deleteAddressEnabled: boolean;
    /** @settingKey MultipleAddresses.MvcPage.AddressTypesThatCanBeDeleted */
    list: AddressTypesThatCanBeDeleted;
}
